.github/workflows/ci-workflow.yaml

```diff
@@ -0,0 +1,23 @@
+name: CI Workflow 
+on: [push, pull_request]
+jobs:
+  build:
+    name: build and test
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
```

Comment:
No need to use a `matrix` strategy if there's only one version of Node being tested against.

Reply:
Included this incase testing multiple versions would be required later. Could remove if not.